apps/prow/prowjobs/kubernetes/k8s.io/periodics.yaml

```diff
@@ -0,0 +1,23 @@
+periodics:
```

Comment:
nit: I am more used to seeing `apps/prow/jobs/{org}/{repo}`

we're gonna want to move this to be top-level or easy to find anyway, so maybe the particular name right now doesn't matter as much

Reply:
We'll update location when I'll add prow self-maintenance jobs.